Please release a 0.5.3 with recent commits

In https://github.com/tomaka/rodio/commit/8eaba65b5df8c0a44949070e59621365d5b248a5
default_endpoint() was exported, and usage examples recommend this pattern in the examples as seen by a potential user.
But this is not yet released, making user (me) confused.

With rodio 0.5.2:

```
warning: use of deprecated item 'rodio::get_default_endpoint'
  --> src\main.rs:10:20
   |
10 |     let endpoint = rodio::get_default_endpoint().unwrap();
   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
``` 

changing:

```
error[E0425]: cannot find function `default_endpoint` in module `rodio`
  --> src\main.rs:10:27
   |
10 |     let endpoint = rodio::default_endpoint().unwrap();
   |                           ^^^^^^^^^^^^^^^^ did you mean `get_default_endpoint`?
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "rodio"
-version = "0.5.2"
+version = "0.5.3"
 authors = ["Pierre Krieger <pierre.krieger1708@gmail.com>"]
 license = "Apache-2.0"
 description = "Audio playback library"

